h/data_tasks/report/create_from_scratch/03_annotations/02_initial_fill.sql

```diff
@@ -25,7 +25,7 @@ SELECT
     -- As we do our partial updates based on updated date, it's good if this
     -- is actually high res timestamp, so we get less overlap
     annotation.updated,
-    deleted,
+    annotation.deleted,
```

Comment:
Need to make it unambiguous that this refers to `annotation.deleted` not `user.deleted`, otherwise Postgres chokes on the query